extensions/amp-story/1.0/amp-story.js

```diff
@@ -1455,6 +1460,39 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @private
+   */
+  openMask_() {
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
```

Comment:
Nit: please move this inside the click listener, so we only get the service when we need it